pkg/cloud/helpers.go

```diff
@@ -43,13 +43,13 @@ func setIntIfPositive(num int64, setFn setInt) {
 	}
 }
 
-func CompressString(str string) (string, error) {
-	buf := &bytes.Buffer{}
-	gzipWriter := gzip.NewWriter(buf)
-	if _, err := gzipWriter.Write([]byte(str)); err != nil {
+func zip(str string) (string, error) {
```

Comment:
nit: should this be?
```suggestion
func gzip(str string) (string, error) {
```